README.md

```diff
@@ -97,22 +97,29 @@ presents the detailed design of the controller.
 
 ## Roadmap
 
-This is a high-level overview of the main priorities for 2023, in expected order of release:
-
-- Cooperative preemption support for workloads that implement checkpointing [#477](https://github.com/kubernetes-sigs/kueue/issues/477)
-- Flavor assignment strategies, e.g. _minimizing cost_ vs _minimizing borrowing_ [#312](https://github.com/kubernetes-sigs/kueue/issues/312)
-- Integration with cluster-autoscaler for guaranteed resource provisioning
-- Integration with common custom workloads [#74](https://github.com/kubernetes-sigs/kueue/issues/74):
-  - Kubeflow (TFJob, MPIJob, etc.)
-  - Spark
-  - Ray
-  - Workflows (Tekton, Argo, etc.)
-
-These are features that we aim to have in the long-term, in no particular order:
-
+High-level overview of the main priorities for 2025:
+- Improve user experience for [MultiKueue](https://kueue.sigs.k8s.io/docs/concepts/multikueue/) - multi-cluster Job disatching, in particular:
+  * sequential attempts to try worker clusters [#3757](https://github.com/kubernetes-sigs/kueue/issues/3757)
+  * log retrieval from worker clusters [3526](https://github.com/kubernetes-sigs/kueue/issues/3526)
+- Improve user experience for [Topology Aware Scheduling](https://kueue.sigs.k8s.io/docs/concepts/topology_aware_scheduling/), in particular:
+  * make Topology-Aware Scheduling compatible with cohorts and preemption [#3761](https://github.com/kubernetes-sigs/kueue/issues/3761)
+  * optimize the algorithm to minimize fragmentation [#3756](https://github.com/kubernetes-sigs/kueue/issues/3756)
+  * better accuracy of scheduling by tighter integration with kube-scheduler [#3755](https://github.com/kubernetes-sigs/kueue/issues/3755)
+  * reduce friction by defaulting the PodSet annotations [#3754](https://github.com/kubernetes-sigs/kueue/issues/3754)
+- Productization of the Kueue dashboard [#940](https://github.com/kubernetes-sigs/kueue/issues/940)
+- Support Hierarchical Cohorts with FairSharing [#3759](https://github.com/kubernetes-sigs/kueue/issues/3759)
+- Improved support for AI inferance, including:
+  * partial preemption of serving workloads [#3762](https://github.com/kubernetes-sigs/kueue/issues/3762)
+  * LeaderWorkerSet support [#3232](https://github.com/kubernetes-sigs/kueue/issues/3232)
+- Progress towards the stable API (v1beta2) [#768](https://github.com/kubernetes-sigs/kueue/issues/768)
+
+Long-term aspirational goals:
```

Comment:
I don't know if I would consider v1beta1 that much difference in stability than v1beta2. Can we add a long-term goal for v1 release of the API?

Reply:
Sure, I pushed an update with the point - if you open such issue I can link it

Reply:
#3476

Reply:
awesome, done